pkg/spdk/engine.go

```diff
@@ -1261,7 +1261,7 @@ func (e *Engine) snapshotOperation(spdkClient *spdkclient.Client, inputSnapshotN
 				e.log.WithError(err).Errorf("WARNING: failed to get the executor for snapshot op %v with snapshot %s, will skip the sync and continue", snapshotOp, inputSnapshotName)
 			} else {
 				e.log.Infof("Requesting system sync %v before snapshot", devicePath)
-				if _, err := ne.Execute(nil, "sync", []string{devicePath}, SyncTimeout); err != nil {
+				if _, err := ne.Execute(nil, "sync", []string{}, SyncTimeout); err != nil {
```

Comment:
Should we also revert the comment and add the finding to it?
https://github.com/longhorn/longhorn-spdk-engine/commit/da2a02e86462fb4f3abac4d0f91f2380ff34053c#diff-2f9990d3c794aaa008aacf70ecd9ae72fc65febf9eee3ea038463cc542e43c36L1100

Reply:
Yes, revert is better. Updated. Thank you.